recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - boto3
     - filelock
     - lightning-cloud ==0.5.64
-    - lightning-utilities >=0.8.0,<0.11.0
+    - lightning-utilities <0.11.0,>=0.8.0
```

Comment:
Would prefer this being sorted the other way, but will just follow grayskull's auto update here.